tests/test_notify_suppliers_of_awarded_briefs.py

```diff
@@ -0,0 +1,192 @@
+import mock
+
+from freezegun import freeze_time
+
+from dmscripts import notify_suppliers_of_awarded_briefs as tested_script
+from dmutils.email.exceptions import EmailError
+from dmutils.email.helpers import hash_string
+
+
+AWARDED_BRIEFS = [
+    {
+        "framework": {
+            "family": "digital-outcomes-and-specialists",
+            "slug": "digital-outcomes-and-specialists-2",
+        },
+        "id": 123,
+        "status": "awarded",
+        "title": "Tea Drinker"
+    },
+    {
+        "framework": {
+            "family": "digital-outcomes-and-specialists",
+            "slug": "digital-outcomes-and-specialists-2",
+        },
+        "id": 456,
+        "status": "awarded",
+        "title": "Cookie Muncher"
+    }
+]
+
+
+EXPECTED_BRIEF_CONTEXT = {
+    'brief_title': "Tea Drinker",
+    'brief_link': 'https://www.preview.marketplace.team/digital-outcomes-and-specialists/opportunities/123'
+}
+
+
+def _get_dummy_brief_response(id_, brief, awarded=False, valid_email=True):
+    brief_response = {
+        "awardedAt": '2018-01-01T23:59:59.999999Z',
+        "briefId": brief['id'],
+        "brief": brief,
+        "id": id_,
+        "respondToEmailAddress": "sore_loser_{}@example.com".format(id_),
+    }
+    if awarded:
+        brief_response['awardedAt'] = '2018-01-01T23:59:59.999999Z'
+        brief_response['status'] = 'awarded'
+        brief_response["respondToEmailAddress"] = "lucky_winner_{}@example.com".format(id_)
+    if not valid_email:
+        brief_response['respondToEmailAddress'] = ""
+    return brief_response
+
+
+def test_create_context_for_brief():
```

Comment:
I'd probably `_` prefix this, if it's not intended to be called from outside this module